cluster/gce/config-test.sh

```diff
@@ -241,6 +241,17 @@ SCHEDULING_ALGORITHM_PROVIDER="${SCHEDULING_ALGORITHM_PROVIDER:-}"
 # Optional: install a default StorageClass
 ENABLE_DEFAULT_STORAGE_CLASS="${ENABLE_DEFAULT_STORAGE_CLASS:-true}"
 
+# Optional: Enable legacy ABAC policy that makes all service accounts superusers.
+if [[ "${E2E_UPGRADE_TEST:-}" == "true" ]]; then
```

Comment:
@krousey @fejta can you verify this value from the [upgrade CI job env files](https://github.com/kubernetes/test-infra/search?q=E2E_UPGRADE_TEST) is visible in this scope?

Other values from those files (like `STORAGE_BACKEND`) seem to be usable in this context, but I wanted to make sure